<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string>MAIN MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(217, 0, 4);</string>
    </property>
    <property name="text">
     <string>DETECT A WEBSITE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>BLACKLIST A WEBSITE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(250, 210, 52);</string>
    </property>
    <property name="text">
     <string>WHITELIST A WEBSITE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);
font: 10pt &quot;Times New Roman&quot;;
background-color: rgb(255, 85, 127);
background-color: rgb(0, 85, 0);
background-color: rgb(107, 213, 0);</string>
    </property>
    <property name="text">
     <string>VIEW BLACKLISTS</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ss4.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
font: 10pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>VIEW WHITELISTS</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
